jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/verification/MathEqualsVerifier.java

```diff
@@ -0,0 +1,35 @@
+package org.jbehavesupport.core.internal.verification;
+
+import org.springframework.stereotype.Component;
+
+import static org.jbehavesupport.core.verification.VerifierNames.MATH_EQ;
+
+@Component
+public final class MathEqualsVerifier extends AbstractVerifier {
+
+    public static final String MESSAGE_PARSE_OK = "value %s (originally written as '%s') is not mathematically equal to %s (originally written as '%s')";
+    public static final String MESSAGE_PARSE_NOK = "couldn't parse either or both value/-s to a number: '%s', '%s'";
+
+    @Override
+    public String name() {
+        return MATH_EQ;
+    }
+
+    @Override
+    public void verify(Object actual, Object expected) {
+        if (actual == expected) {
+            return;
+        }
+
+        try {
+            Double actualDouble = Double.valueOf(actual.toString());
```

Comment:
More of a question, but cant the toString hurt us in some way? Maybe conversionService.convert might be a bit safer in this case?

Reply:
@paveljandejsek  You're probably right about the conversionService. I'll update it accordingly.

Regarding the other MATH_ versions, the only one I can think of for now is MATH_NEQ (eg. `1.0 NEQ 1 == true`, but `1.0 MATH_NEQ 1 == false`) - @xsmrcek what do you think?